src/arch/x86_64/paging/mod.rs

```diff
@@ -131,6 +131,18 @@ pub fn initialize_pagetables(
 				.unwrap()
 		};
 	}
+
+	// SAFETY: Should be fine, as long as the RAM start does not explicitly conflict with this.
+	// Used for MmioWrite-based hypercalls.
+	//
+	// FIXME: Only map this page if the uhyve-interface version is at least higher than 2.
```

Comment:
Before adding a commit that couples the uhyve-interface version to this function, it might be worth investigating if this is also required on macOS (-> will add test cases first).

If non-relocatable images with uhyve-interface v2 become a thing (will they?), they will not work with this either.